### PR TITLE
Updated release section with bug fixes

### PR DESCRIPTION
**Pull request by cpaul92**

### CODE REVIEW

File: docs/api-documentation/V1.18/index.md [hunk 1]
```diff
@@ -2768,6 +2768,8 @@ Updated Work API with the following changes:
   <li>Planned permits can be assessed at any stage, provided they haven't been assessed or cancelled</li>
   <li>Updated the <code>PUT /works/{workReferenceNumber}/permits/{permitReferenceNumber}/assessment</code> endpoint to now accept the additional values <code>reasonable_period_end_date</code> and <code>is_duration_challenged</code></li>
   <li>The <code>PermitResponse</code> has been updated to return <code>is_duration_challenged</code></li>
+  <li>Updated the <code>DELETE /files/{fileId}</code> to return a precondition failed error if the file has already been associated with a work</li>
```
Comment (andrew-js-wright) [inline docs/api-documentation/V1.18/index.md]: Can you reword this to something like `Fixed a defect with the <code>DELETE /files/{fileId}</code> endpoint so it now correctly returns a precondition failed error if the file has already been associated with a work`

Just to make it clear that it was always intended to work like this and this isn't a wholesale breaking change.

Comment (cpaul92) [reply docs/api-documentation/V1.18/index.md]: Updated

File: docs/api-documentation/V1.18/index.md [hunk 2]
```diff
@@ -2768,6 +2768,8 @@ Updated Work API with the following changes:
   <li>Planned permits can be assessed at any stage, provided they haven't been assessed or cancelled</li>
   <li>Updated the <code>PUT /works/{workReferenceNumber}/permits/{permitReferenceNumber}/assessment</code> endpoint to now accept the additional values <code>reasonable_period_end_date</code> and <code>is_duration_challenged</code></li>
   <li>The <code>PermitResponse</code> has been updated to return <code>is_duration_challenged</code></li>
+  <li>Fixed a defect with the <code>DELETE /files/{fileId}</code> endpoint so it now correctly returns a precondition failed error if the file has already been associated with a work</li>
+  <li>Added <code>inspection_agreed_site_compliance</code> enum to AuditEvent</li>
```
Comment (andrew-js-wright) [inline docs/api-documentation/V1.18/index.md]: Same here, it would be good to call out the endpoint it affected and the fact that we're fixing a defect.
